panoptes_aggregation/tests/extractor_tests/test_question_extractor.py

```diff
@@ -87,3 +87,37 @@
     kwargs={'task': 'T1'},
     test_name='TestSingleTwoTasksKey'
 )
+
+single_pluck_classification = {
+    "id": 359841171,
+    "annotations": [{
+        "task": "T0",
+        "value": "0"
+    }],
+    "subject": {
+        "id": 68561182,
+        "metadata": {
+            "filename": "uds_H.gf19767.png",
+            "uber_flag_digit": "4",
+            "is_gold_standard": "False"
+        }
+    }
+}
+
+single_pluck_keys = '{\
+    "gold_standard": "subject.metadata.is_gold_standard",\
+    "true_value": "subject.metadata.uber_flag_digit"\
+}'
```

Comment:
This should not be a string given the way this is currently set up, pass it in as a dictionary.